portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java

```diff
@@ -2286,4 +2322,25 @@ protected String getSkin(String skin)
 		return CSSUtils.adjustCssSkinFolder(skin);
 	}
 
+	private boolean impersonating() {
+
+	Session s = SessionManager.getCurrentSession();
+	String  userId = s.getUserId();
+	UsageSession session = (UsageSession) s.getAttribute(UsageSessionService.USAGE_SESSION_KEY);
+	if (session != null)
+	{
```

Comment:
can you make the brace styling consistent with the block further up? That is, on the same line, not a new line?